runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -73,10 +62,16 @@ impl GasCost {
     /// Creates `GasCost` out of raw numeric value of gas. This is required mostly for
     /// compatibility with existing code, prefer using `measure` instead.
     pub(crate) fn from_gas(raw: Ratio<u64>, metric: GasMetric) -> GasCost {
```

Comment:
After this PR is merged, it makes sense to revisit `from_gas` call-sites, perhaps this can be finally removed.